### PR TITLE
Update bootstrap-v3 for v3.4.0

### PR DESCRIPTION
**Pull request by XhmikosR**
**WIP until we make the release.**

Let me know if there's something wrong or something that can be simplified :)

PS. I assume this won't interfere with the 4.x index, right? Since basically we follow the `docs/XX` scheme.

### CODE REVIEW

File: configs/bootstrap-v3.json [hunk 1]
```diff
@@ -1,13 +1,14 @@
 {
   "index_name": "bootstrap-v3",
   "start_urls": [
-    "https://epic-goldstine-3a9d25.netlify.com/"
+    "http://getbootstrap.com/docs/3.4/",
+    "https://getbootstrap.com/docs/3.4/"
```
Comment (s-pace) [inline configs/bootstrap-v3.json]: Can we only stick to the https pages?

**Comment by s-pace** [comment]
> PS. I assume this won't interfere with the 4.x index, right? Since basically we follow the docs/XX scheme.

You are write, we will not crawl pages that aren't matching https://getbootstrap.com/docs/3.4/*

**Comment by XhmikosR** [comment]
@s-pace if all goes well we'll release this later today. :)

**Comment by s-pace** [comment]
@XhmikosR great news! Let me know when you are ready. We will merge it as soon as possible

**Comment by XhmikosR** [comment]
@s-pace: we released this today!

**Pull request merged**